presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -60,45 +63,56 @@
     @BeforeTestWithContext
     public void setup()
     {
-        tableName = "alice_owned_table";
         aliceExecutor = connectToPresto("alice@presto");
         bobExecutor = connectToPresto("bob@presto");
 
-        aliceExecutor.executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
-        aliceExecutor.executeQuery(format("CREATE TABLE %s(month bigint, day bigint)", tableName));
+        aliceExecutor.executeQuery(format("DROP TABLE IF EXISTS %s", TABLE_NAME));
```

Comment:
It's best to have a different tablename for each test, we may run tests in parallel soonish, same about the view names. Either each test needs to have a randomly generated tablename, and you'd have to store the tablename as a field, or you need to do try/finally around the code in each test.

I realize this is an extension to what I asked for before, but I didn't know that each test had tables with the same name.